MapboxDirections/MBVisualInstructionComponent.swift

```diff
@@ -34,20 +34,24 @@ open class VisualInstructionComponent: NSObject, NSSecureCoding {
     
     /**
      :nodoc:
-     `Bool` whether the `VisualInstructionComponent` should be visualized as a delimiter between two text components.
+     The type of the `VisualInstructionComponent`. Used for influencing how the compomenent is rendered.
```

Comment:
Typo: component. Also, it isn’t necessary to link to `VisualInstructionComponent`, since it’s the containing class:

> The type of visual instruction component. You can display the component differently depending on its type.